README.md

```diff
@@ -24,22 +31,54 @@ Proteomics data: [PXD043300](https://proteomecentral.proteomexchange.org/cgi/Get
 
 Phosphoproteomics data: [PXD043297](https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD043297)
 
-### Organize the data:
+### Organize Data
 
-All data should be organized in a datasets using the following structure
-`datasets/consortium/<model>/01_received_data/<omic>`
-and a metadata directory
-`datasets/consortium/<model>/01_received_data/cohort`.
+All data should be organized in datasets using the following structures
+
+```
+datasets/
+    consortium/
+        <model>/
+            01_received_data/
+                <omic>/
+            cohort/
+```
 
 The pipeline expects `fastq.gz` files for the sequencing data und `txt` files for the proteomics data and `csv` files for the phosphoproteomics data.
 
-Please, use [DVC](https://dvc.org/) to see which exact files names are required:
+Please, use [DVC](https://dvc.org/) to see which exact file names are required:
+
+```bash
+dvc status srna_organize_samples proteomics_organize_samples phosphoproteomics_organize_samples rnaseq_nextflow
+```
+
+
+## Reproducing Results
+
+To reproduce the analysis results, execute the following command:
+
+```bash
+dvc repro
+```
+
+This command will run the predefined pipelines to process and analyze the data according to the methodology described in the associated publication.
+
+## Contributing
+
+We welcome contributions to enhance the reproducibility and scope of the analysis. Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduct and the process for submitting pull requests.
+
```

Comment:
This part should be done before the downloading, otherwise it will be outside of the maxomod directory.